attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>In the solution and expected results you expect hidden_states @ previous_state.T but in your formulas you use hidden_states.T @ previous_state</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Whoops! Nice catch thank you, fixing it now. </p>

Reply:
<p>Very good catch thank you! Code worked by chance just due to the word embeddings examples.</p>

Reply:
<p>@<a href="https://github.com/hereismari" target="_blank">hereismari</a>, just agree if you now agree with what I did. Can also chat over slack, drew everything out now again to also make sure, but mistakes slip in very easy.</p>